src/api/app/assets/stylesheets/webui/responsive_ux.scss

```diff
@@ -65,6 +65,23 @@ body.responsive-ux {
       &.center { margin-left: calc(38% + .75rem); }
       &.left { left: .5rem; }
     }
+  }
+}
 
+@include media-breakpoint-between(xs, sm) {
+  body.responsive-ux {
+    #notifications-filter-desktop {
+      &.show { border-top: 1px solid $gray-300;
```

Comment:
The alignment/indentation is a bit off in this file. This should be fixed.

Reply:
Thanks for feedback, I fixed the the indentation :)